tests/integration/transformers/test_categorical.py

```diff
@@ -348,3 +348,26 @@ def test_label_encoder_order_by_alphabetical():
     expected = pd.DataFrame([1, 3, 4, 2, 0], columns=['column_name.value'])
     pd.testing.assert_frame_equal(transformed, expected)
     pd.testing.assert_frame_equal(reverse, data)
+
+
+def test_custom_label_encoder():
+    """The the CustomLabelEncoder.
```

Comment:
Should be "Test the CustomLabelEncoder"

Nit: The comment could be more descriptive